nextflow_schema.json

```diff
@@ -200,13 +228,11 @@
                 },
                 "skip_subsample_region": {
                     "type": "boolean",
-                    "default": false,
```

Comment:
Why remove the default here and below? 

I had problems with the linting before if that is the case, seems to be resolved now.

Reply:
Done!